common/image.go

```diff
@@ -91,7 +92,7 @@ func (i ImageRef) RelativeWidth(h measurement.Distance) measurement.Distance {
 }
 
 // ImageFromFile reads an image from a file on disk. It doesn't keep the image
-// in memory and only reads it to determine the format and size.  You can also
+// in memory and only reads it to determine the format and size. You can also
 // construct an Image directly if the file and size are known.
```

Comment:
Add
// DEPRECATED: This function is deprecated and will be removed in a future major version (v2).  Use ImageFromStorage instead.

Reply:
I didn't suppose this function to be deprecated, it is used to add images from files by clients (and I initially wanted to make ImageFromStorage internal for using it only for temporary reasons), do you think that all images should be added by tempstorage.Add and then common.ImageFromStorage?

Reply:
Since ImageFromStorage uses diskstore by default this behavior is unaffected.  If they switch to memory store, it is likely that they want to use that behavior for other things as well.  We can keep both, maybe add a note to point to the ImageFromStorage also.  // NOTE: See also ImageFromStorage.

Reply:
Added a note about ImageFromStorage